.stylelintrc.json

```diff
@@ -28,5 +28,5 @@
       ],
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
I don't agree with you cause they said to concentrate on functionality not styleing

Reply:
Great work dear @richie1988 but it is just an optional requirement to make your readme more professional 🤔 so you can do it anytime you want 😄